pkg/proxy/config/config.go

```diff
@@ -110,8 +104,6 @@ func (c *EndpointsConfig) Run(stopCh <-chan struct{}) {
 		klog.V(3).Infof("Calling handler.OnEndpointsSynced()")
 		c.eventHandlers[i].OnEndpointsSynced()
 	}
-
-	<-stopCh
```

Comment:
If you remove the blocking on stop channel, then you shouljd also remove the `defer klog.Info("Shutting down XXXX config controller")` lines for services and endpoints too, since those will now be called right after sync when the function returns, rather than when the process exits.

Reply:
Also, you can remove stopCh from both endpoints/services now because it's no longer used.

Reply:
> If you remove the blocking on stop channel, then you shouljd also remove the defer klog.Info("Shutting down XXXX config controller") lines for services and endpoints too, since those will now be called right after sync when the function returns, rather than when the process exits.

Thanks, I fixed this.

> Also, you can remove the stopCh argument from both endpoints/services Run() now because it's no longer used.

stopCh is used as argument for WaitForCacheSynced. Currently, wait.Neverstop is passed, but I think that it is not bad for the caller of Run () to decide which channel to pass to WaitForCacheSync.

Reply:
@mrkm4ntr ah right, I missed that for stopCh. Nevermind about that bit then.